specification/search/data-plane/Microsoft.Azure.Search.Service/stable/2019-05-06/searchservice.json

```diff
@@ -4519,6 +4519,10 @@
         "@odata.type": {
           "type": "string"
         },
+        "name": {
+          "type": "string",
+          "description": "The name of the skill which uniquely identifies the skill within the skillset. Skills with no skill name defined will be given a default name of its index in the skillset array."
```

Comment:
@jadrefke The word "skill" is used here an awful lot. Plus, "skillset" array is a misnomer -- the array is called "skills", right? How about this:

> The name of the skill which uniquely identifies it within the skillset. A skill with no name defined will be given a default name of its 0-based index in the skills array.